README.md

```diff
@@ -70,14 +67,13 @@ $middleware->add(new SocialAuthMiddleware([
     'finder' => 'all',
     // Fields.
     'fields' => [
-        'email' => 'email',
         'password' => 'password'
     ],
     // Session key to which to write identity record to.
     'sessionKey' => 'Auth.User',
     // The methods in user model which should be called in case of new user.
```

Comment:
Statement needs to be updated since the callback is no longer called for just new user.